nautobot/extras/models/relationships.py

```diff
@@ -249,6 +261,14 @@ class Relationship(BaseModel, ChangeLoggedModel, NotesMixin):
         default=RelationshipTypeChoices.TYPE_MANY_TO_MANY,
         help_text="Cardinality of this relationship",
     )
+    required_side = models.CharField(
+        max_length=12,
+        choices=RelationshipRequiredSideChoices,
+        default=RelationshipRequiredSideChoices.NEITHER_SIDE_REQUIRED,
+        help_text="Objects on the specified side MUST implement this relationship. Not permitted for symmetric relationships.",
```

Comment:
Probably need `clean()` logic to raise an error if (symmetric && required)

Reply:
Ah yes!